package.json

```diff
@@ -100,16 +100,37 @@
         "configuration": {
             "title": "Fracas",
             "properties": {
+                "vscode-fracas.general.projectDir": {
+                    "type": "string",
+                    "default": "C:\\proj\\ws\\tdp1\\wslib",
```

Comment:
Some content devs (mostly artists and animators, I think) have a different path to the project -- how will this affect them?

Reply:
They'll need to enter the correct path in VS Code user settings
![image](https://user-images.githubusercontent.com/5162625/148627279-9b46b960-a81c-4252-979a-31edbf84ab0e.png)